CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+DBLINK Version 0.2.1 (16 Dec 2022)
```

Comment:
Please stop using `CHANGELOG.md`. You should 
1. Delete the `CHANGELOG.md` in this pull request
2. Merge this pull request
3. Create a new release in https://github.com/vertica/dblink/releases

Reply:
@roypaulin can you please help @mfelici to do it?

Reply:
Yes, we will go through that when he comes back from holidays. I think the release stuff will be after the tests are added.